TappNetwork/home-visiting-app-web
# reset password button and page created

# Details

This PR is for the front and back end changes for the home visiting apps password reset feature.

# Steps to test changes

1. Pull down this branch in both repos
2. Launch mailhog and configure backend .env if necessary. 
3. go to sign in page of HV App front end and request password reset
4. click link in email sent to go to password reset page
5. test client side validation by giving short and mismatching passwords
6. test server side validation by giving an email different than the one the link was sent to
7. Change password and confirm new passwords works to log in to app

# Known Issues
I am unable to get the alert modal to display when request is successful. The code in my .then closure for both the link request and reset request is never executed. If I am able to fix this, I want to put the redirect for the reset into the success closure, so that user has a chance to correct their input if they make a typo on their email. 

# Issues Addressed

[2r1j6x6](https://app.clickup.com/t/2r1j6x6)

# Screenshots
<img width="649" alt="Screen Shot 2022-09-13 at 2 45 23 PM" src="https://user-images.githubusercontent.com/29612767/190005806-c380d805-5c0d-47f0-abb8-b85c168e3265.png">
<img width="1052" alt="Screen Shot 2022-09-13 at 2 45 35 PM" src="https://user-images.githubusercontent.com/29612767/190005820-f472febb-38f1-496e-8520-811ac7af80dc.png">
